src/shortcodes/index.test.js

```diff
@@ -0,0 +1,29 @@
+/**
```

Comment:
purely for my own edification -why do you include the test files throughout the repo, rather than a `/test` directory?  do you prefer this style for some reason?

Reply:
I prefer it because it makes it a lot easier to locate tests for a given file (and makes it more obvious if tests are missing).